tests/src/JIT/HardwareIntrinsics/X86/Shared/GenerateTests.csx

```diff
@@ -1053,6 +1062,12 @@ private static void ProcessInput(StreamWriter testListFile, string groupName, (s
 
     var suffix = "";
 
+    if (input.templateFileName == "SimpleUnOpConvTest.template" || input.templateFileName == "SimdScalarUnOpConvTest.template" )
```

Comment:
Wasn't this being added by a different PR?

Reply:
Yes, I am blocked in this PR by #19677, which also blocks work on #19670. Therefore decided to work further and wait for #19677 review and merge to rebase on top of #19677. These PRs also block further work on conversions.